lib/mindwendel_web/live/brainstorming_live/show.ex

```diff
@@ -85,4 +85,8 @@ defmodule MindwendelWeb.BrainstormingLive.Show do
   def handle_event("sort", %{"id" => id}, socket) do
```

Comment:
Porposal: rename to sort_by_likes or something like that

Reply:
Done.